mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -479,10 +479,12 @@ def delete_experiment(self, experiment_id):
     def _mark_run_deleted(self, session, run):
         self._check_run_is_active(run)
         run.lifecycle_stage = LifecycleStage.DELETED
+        run.delete_time = int(time.time() * 1000)
```

Comment:
Nit: Can we use `deleted_time` instead?

```suggestion
        run.deleted_time = int(time.time() * 1000)
```

Reply:
:ok_hand: